jobs/uaa-customized/templates/web/login.html

```diff
@@ -118,9 +118,21 @@ <h2 class="govuk-heading-m">Google</h2>
         </a>
       </p>
 
-      <th:block th:each="oauthLink : ${oauthLinks}" th:inline="html">
-        <!-- [(${oauthLink.value})] [(${oauthLink.key})] -->
-      </th:block>
+      <h2 class="govuk-heading-m">Microsoft</h2>
+
+      <p class="govuk-body">
+        This option is open to everyone with a Microsoft account, who has
+        enabled single sign-on using the admin panel.
```

Comment:
I think this needs rephrasing–the comma splits it into two fragments, neither of which quite makes sense. It's truer to say this option is open to everyone who has enabled Microsoft single sign-on using the admin panel. Maybe Jon could be helpful here?

Reply:
Yes, agreed. Jon's input would be welcome wherever there are words to be read by a user.

Reply:
Discussed it with Jon and he thinks "remove the comma". I've pushed a commit to do that (and from the google one too, since that sentence has the same structure).